vault/changelog.release.2021-12-28.md

```diff
@@ -0,0 +1,92 @@
+---
+id: N1Mn5p6O2tVVkdDM0Ys1w
+title: '0.75'
+desc: ''
+updated: 1640719141612
+created: 1640715852170
+---
+
+Dendron 0.75 has sprouted  🌱
+
+This is the last release of the year!
+
+When deleting notes, did you just break some links elsewhere? Now Dendron makes sure to give you a heads up, about any links that will break, with the latest update to the `Dendron: Delete Node` command.
```

Comment:
Lets tie this in with the new year. Change to ->

Make any mistakes this year you wish you could take back? Moving forward, if those mistakes involve accidentally deleting a note, Dendron will now ask for confirmation when deleting as well as show a preview of all links that will be broken from the operation.